services/hooks/src/api.js

```diff
@@ -534,6 +538,7 @@ builder.declare({
   route: '/hooks/:hookGroupId/:hookId/trigger/:token',
   name: 'triggerHookWithToken',
   input: 'trigger-hook.yml',
+  scopes: 'hooks:trigger-hook:<hookGroupId>/<hookId>',
```

Comment:
This API method is intended to be used as a webhook, so I think `scopes: null` would make sense.